numba/cuda/cudadrv/enums.py

```diff
@@ -115,6 +115,15 @@
 CU_MEMHOSTREGISTER_DEVICEMAP = 0x02
 
 
+# If set, managed memory is accessible from all streams on
+# all devices.
+CU_MEM_ATTACH_GLOBAL = 0x01
+
+# If set, managed memory is not accessible from all streams on
```

Comment:
Can the comment explain to what it is accessible from if this is set?

Reply:
I have clarified this comment; the original text was from the CUDA driver API documentation, which was admittedly not helpful. For those curious, more information about this can be found in the API documentation for [cuStreamMemAttachAsync](https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__STREAM.html#group__CUDA__STREAM_1g6e468d680e263e7eba02a56643c50533) and [cuMemAllocManaged](https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__MEM.html#group__CUDA__MEM_1gb347ded34dc326af404aa02af5388a32).